content/en/hosting/4.x/production/google-cloud/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Production Hosting CHT 4.x - Google Cloud Platform"
+linkTitle: "Google Cloud Platform"
+weight: 10
+aliases:
+   - /hosting/4.x/production/docker/google-cloud/
+description: >
+   Production Hosting CHT 4.x - Google Cloud Platform
+---
+This for a step by step to migrate or deploy CHT 4.x instance and is the recommended solution  on Google Cloud Platform.
+
+## Prerequisites
+
+Before you can start using  **Google Cloud Platform (GCP)** , you need to meet certain requirements
+
+1. **Google Account:** a valid Google account (Gmail or Google Workspace)
+2. **Enable Billing & Free Trial ($300 Credit)**: a valid **credit card** (identity verification)
+3. **Create a GCP Project**: each resource in GCP must belong to a **project**
+
+## Template resources, changes required and some considerations
+
+Let's start writing out the detail as to what and why, and gathering those links on info so the cht-docs PR will be easier.
+For example, what is the main docker image constraint (base image selection)? Why don't we want to use google-optimized container images, or Amazon-optimized container images? Why do we need to modify u-limit (and link to the relevant couchdb performance doc)
+
+## **Recommendations and considerations**
+
+### Why not to use optimized container images and why modify Ulimit ?
+
+On a system with many databases or many views, CouchDB can very rapidly hit the Ulimit, and this is not possible with either Google-optimized container images or Amazon-optimized container images. [More details for couchDB performance](https://docs.couchdb.org/en/stable/maintenance/performance.html)
+
+This is why we use Ubuntu with containerd (ubuntu_containerd)
+
+### Cluster creation
+
+#### Zonal cluster
+
+* Pros: Less costly
+* Cons: Single control plane in single zone
+
+#### Regional cluster
+
+* Pros: Multiple control plane replicas across multiple compute zones
+* Cons: High cost
+
+### VPC Network
```

Comment:
Let's add more details that resolve the original questions from Muso's request directly.

- Add our CIDR
- Small description to why VMs that run containers are in the private subnet

- Add a section for bastion/ssh-jumpbox setup. That's how we will troubleshoot underlying VM issues. We will want to add a VM to the public subnet that includes routes to the private servers. After tomorrow' session, we can fill that out with exact config details or photos showing how to deploy bastion. There's already existing work in cht-core we can perhaps utilize and link to.